x-pack/plugin/ccr/qa/multi-cluster-with-security/src/test/java/org/elasticsearch/xpack/ccr/FollowIndexSecurityIT.java

```diff
@@ -111,6 +111,53 @@ public void testFollowIndex() throws Exception {
         }
     }
 
+    public void testCreateAndFollowIndex() throws Exception {
```

Comment:
I do think that two separate tests would be better. One that tests against an index with appropriate permissions, and one that tests against an index without appropriate permissions. Most of the code can be shared between the two methods, it is only about what is asserted at the end that needs to change.

Reply:
Agreed, I'm going to change this.